backend/backend.go

```diff
@@ -17,30 +17,30 @@ var backends = make(map[string]Factory)
 // Factory is a function that returns a MetricSender.
 type Factory func(v *viper.Viper) (MetricSender, error)
 
-// MetricSender represents a backend
+// MetricSender represents a backend.
```

Comment:
Well, no, it was me who just decided to fix the style issue. I hope that is ok - it is recommended to put dot at the end.